.grit/patterns/jest_array_containing.md

```diff
@@ -1,4 +1,6 @@
-# Consolidate Jest arrayContaining expectations
+---
+title: Consolidate Jest arrayContaining assertions
```

Comment:
Are we not properly picking up titles from the headline? We shouldn't *require* frontmatter.

Reply:
I only changed it to match the styling of every other `.md` file, since they all use the frontmatter.